docs/installation.md

```diff
@@ -218,19 +218,234 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
 
 ------
 
-### Anaconda
+## Conda (Miniconda or Anaconda)
 
-Freqtrade can also be installed using Anaconda (or Miniconda).
+Freqtrade can also be installed with Miniconda or Anaconda.
+Conda is a package manager and virtual environment manager in one.
+The purpose of installing Conda (Miniconda or Anaconda) is to automatically prepare and manage the extensive library dependencies of the Freqtrade program.
 
-!!! Note
-    This requires the [ta-lib](#1-install-ta-lib) C-library to be installed first. See below.
+You can start with any clean distribution of Debian-based Linux distributions, which includes Ubuntu Linux.
+For other popular Linux distributions, see: https://distrowatch.com/ or https://www.ubuntupit.com/best-debian-based-linux-distributions/
+
+Installation requirement: Your Linux is a Debian based distribution with bash terminal
+
+To get know more about Conda check : https://linuxnetmag.com/miniconda-vs-anaconda/
+
+Shortly : Conda < Miniconda < Anaconda
+
+https://www.dunderdata.com/blog/anaconda-is-bloated-set-up-a-lean-robust-data-science-environment-with-miniconda-and-conda-forge
+
+The difference lies in the number of packages that need to be installed upfront: how heavy/light the installation is. Difference is like ~1:8. It is recommended to install Miniconda instead of Anaconda.
+
+#### 1. Instal Conda
+
+##### Download conda file:
+
+installation file of chosen distribution you can find here
+
+https://docs.conda.io/en/latest/miniconda.html
+
+##### Verify file integrity with SHA256 checksum:
+
+official SHA256 checksum can be found here
+
+https://docs.conda.io/en/latest/miniconda_hashes.html
+
+Navigate to the directory with your file, and run SHA256 checksum of your file, check with following command
+
+```bash
+sha256sum ./Miniconda3-number-of-the-package
+```
+
+both checksum suppose to match
+
+##### Install conda:
+
+While in directory with your file, install conda
+
+```bash
+bash ./Miniconda3-number-of-the-package
+```
+
+Confirm with yes all questions
+
+After installation, it is mandatory to turn your terminal OFF and ON again.
+
+CTRL + D (close terminal), CTRL + ALT + T (open terminal)
+
+#### 2. Prepare Conda environment
 
-``` bash
-conda env create -f environment.yml
+
+enter the conda base environment:
+
+```bash
+conda activate base
+```
+
+If you want, you can prevent the (base) conda environment from being activated automatically as well.
+
+```bash
+conda config --set auto_activate_base false
+```
```

Comment:
This seems to be very platform specific - under windows, you can open the conda prompt and base is activated immediately.

Again (as above), this will be in the conda docs, so no point in repeating this here.
Please keep it to a minimum, assuming the user is familiar with conda already.

Reply:
Script is changed a bit. 
I would rather not assume that everybody knows conda,

Reply:
neither would i :) 
i'm sure conda has a nice "getting started" documentation where people can learn how to use it.

Reply:
And to this place, our documentation can refer, 
and can also strengthen the focus on certain paragraphs.